tests/conftest.py

```diff
@@ -348,6 +354,11 @@ def table_schema_with_all_types() -> Schema:
     )
 
 
+@pytest.fixture(params=["abfss", "wasbs"])
```

Comment:
```suggestion
@pytest.fixture(params=["abfs", "abfss", "wasb", "wasbs"])
```

should we include the other 2 variants?